[2024-07-22T02:31:13.780+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-04-02T00:00:00+00:00 map_index=2 [queued]>
[2024-07-22T02:31:13.861+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-04-02T00:00:00+00:00 map_index=2 [queued]>
[2024-07-22T02:31:13.885+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:31:13.910+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:31:13.912+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:31:14.014+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-04-02 00:00:00+00:00
[2024-07-22T02:31:14.025+0000] {standard_task_runner.py:55} INFO - Started process 731 to run task
[2024-07-22T02:31:14.037+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-04-02T00:00:00+00:00', '--job-id', '131', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmp0xv1r4c6', '--map-index', '2']
[2024-07-22T02:31:14.045+0000] {standard_task_runner.py:83} INFO - Job 131: Subtask add_one
[2024-07-22T02:31:14.148+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 131 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-04-02T00:00:00+00:00/task_id=add_one/map_index=2/attempt=1.log'; 731)
[2024-07-22T02:31:14.160+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:31:14.201+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
